examples/lightning_base.py

```diff
@@ -139,7 +125,9 @@ def _feature_file(self, mode):
         )
 
     @staticmethod
-    def add_model_specific_args(parser, root_dir):
+    def add_model_specific_args(parent_parser):
```

Comment:
Will probably need to parse temp args here to import `xm`. Otherwise the style checks will continue to fail on undefined variable. @williamFalcon - thoughts? Is `xm` still needed here for TPU?